main/resource-access.bs

```diff
@@ -75,6 +75,291 @@ Issue: Write Linked Data Platform section.
+privileges per the [ACL inheritance
+algorithm](https://github.com/solid/web-access-control-spec#acl-inheritance-algorithm)
+on the resource directly associated with it.
+
+An ACL metadata resource MUST NOT be deleted unless the resource directly
+associated with it is deleted.
+
+A Solid server SHOULD sanity check ACL metadata resources upon creation or
+update to restrict invalid changes, such as by performing shape validation
+against authorization statements therein.
+
+#### Resource Description #### {#rm-description}
+
+Issue: Identify any resource description metadata parameters that MUST be
+recognized by all implementations.
+
+Resource description is a general mechanism to provide descriptive metadata for
+a given resource. It MUST be supported as a resource metadata type by Solid
+servers.
+
+The Descriptive metadata resource directly associated with a given resource is
+discovered by the client via the `"rel=describedby"` `Link` relation in a `Link`
+header. Conversely, the resource being described by a Descriptive metadata
+resource is discovered by the client via the `rel="describes"` `Link` relation
+in a `Link` header.
+
+A given Solid resource MUST NOT be directly associated with more than one
+Descriptive metadata resource.
+
+To create or modify a Descriptive metadata resource, a given
+[acl:agent](https://github.com/solid/web-access-control-spec#describing-agents)
+MUST have
+[acl:Write](https://github.com/solid/web-access-control-spec#aclcontrol)
+privileges per the [ACL inheritance
+algorithm](https://github.com/solid/web-access-control-spec#acl-inheritance-algorithm)
+on the resource directly associated with it.
+
+To discover or read a Descriptive metadata resource, an
+[acl:agent](https://github.com/solid/web-access-control-spec#describing-agents)
+MUST have
+[acl:Read](https://github.com/solid/web-access-control-spec#aclcontrol)
+privileges per the [ACL inheritance
+algorithm](https://github.com/solid/web-access-control-spec#acl-inheritance-algorithm)
+on the resource directly associated with it.
+
+#### Shape Validation #### {#rm-shape}
+
+Issue: Detail the composition of a shape metadata resource, including the shape
+language, shape url, and any additional parameters to be used in shape
+validation by the server implementation.
+
+Shape Validation ensures that any data changes in a given resource conform to an
+associated [[!SHACL]] or [[!SHEX]] data shape. It MUST be supported as a
+resource metadata type by Solid servers.
+
+The Shape validation metadata resource directly associated with a given resource
+is discovered by the client via the `rel=http://www.w3.org/ns/solid/terms#shape`
+`Link` relation in a `Link` header. Conversely, the resource being described by
+a Shape validation metadata resource is discovered by the client via the
+`rel=describes` `Link` relation in a `Link` header.
+
+A given Solid resource MUST NOT be directly associated with more than one
+Descriptive metadata resource.
```

Comment:
duplicate?

Reply:
Sure, but that's a tooling feature (copy ACLs from X)